### PR TITLE
feat(codelab/kayenta-workship): Add services and supporting artifacts.

### PR DESCRIPTION
**Pull request by duftler**
Will resolve the duplication with gke-source-to-prod later. Have not customized the README either.

### CODE REVIEW

**Comment by duftler** [comment]
@lwander 1st pass is ready for a review.

File: codelabs/gke-base/install/manifests.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ spec:
     spec:
       containers:
       - name: halyard-daemon
-        image: gcr.io/spinnaker-marketplace/halyard:stable
+        image: gcr.io/spinnaker-marketplace/halyard@sha256:5c34d37b7e6fbcc1776d2cd9385ebaf063a7b6d670abacb0fa6c2b5886e5d47f
```
Comment (lwander) [inline codelabs/gke-base/install/manifests.yml]: Intentional to pin this?

File: codelabs/gke-base/install/enable_apis.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+bold "Enabling Cloud Build API..."
+
+gcloud services --project $PROJECT_ID enable cloudbuild.googleapis.com
+
+
+bold "Enabling Kubernetes Engine API..."
+
+gcloud services --project $PROJECT_ID enable container.googleapis.com
```
Comment (lwander) [inline codelabs/gke-base/install/enable_apis.sh]: Do you think we'll need more APIs in other codelabs? If so, maybe we can drop a note to move this list of apis to the `properties` file in a future change.

File: codelabs/gke-kayenta-workshop/services/backend/main.go [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+	"google.golang.org/api/monitoring/v3"
+)
+
+var count = 1
+var lastTime = time.Now()
+
+// Started with: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/monitoring/custommetric/custommetric.go
+func writeTimeSeriesValue(now string, s *monitoring.Service, metricType string, metricValue int64) error {
+	project_id, err := gce.ProjectID()
```
Comment (lwander) [inline codelabs/gke-kayenta-workshop/services/backend/main.go]: Worth checking these errors?

File: codelabs/gke-base/install/manifests.yml (hunk 1 above)
Comment (duftler) [reply codelabs/gke-base/install/manifests.yml]: At the time, it was the version that had the fix we needed. Looks like that version is now also marked `stable`. Fixed.

File: codelabs/gke-base/install/enable_apis.sh (hunk 2 above)
Comment (duftler) [reply codelabs/gke-base/install/enable_apis.sh]: Done.

File: codelabs/gke-kayenta-workshop/services/backend/main.go (hunk 3 above)
Comment (duftler) [reply codelabs/gke-kayenta-workshop/services/backend/main.go]: Probably. There are a bunch of things I'd like to change in this driver. I dropped a TODO.

**Pull request merged**